source/aws_signing.c

```diff
@@ -1707,6 +1707,90 @@ static int s_build_canonical_request_hash(struct aws_signing_state_aws *state) {
     return result;
 }
 
+static int s_build_canonical_request_event(struct aws_signing_state_aws *state) {
+    int result = AWS_OP_ERR;
+
+    struct aws_byte_buf *dest = &state->string_to_sign_payload;
+
+    /* previous signature + \n */
```

Comment:
trivial: For long functions like this, it's nice to block out the major sections with bigger comment blocks. And it's nice to call back to the larger comment above...
```suggestion
    /*
     * Hex(priorSignature) + "\n"
     * 
     * Fortunately, the prior signature is already hex.
     */
```